tests/SecurityGroup.py

```diff
@@ -15,7 +15,8 @@ def __init__(self, nonParticipants, contractAccount, defaultNode=None, minAddRem
         self.participants = []
         self.contractAccount = contractAccount
         assert len(nonParticipants) > 0
-        self.nonParticipants = copy.copy(nonParticipants)
+        # copy over all the running processes
+        self.nonParticipants = [x for x in nonParticipants if x.pid]
```

Comment:
why do you need this? this will make more difficult to deal with `unstartedNodes`

Reply:
This is needed so that the killed bios node (killed because it has a mesh connection to everything) to keep it from failing the checks for the nonParticipants.  I guess we could put checks in the code to check for node.pid before:
moving it to the participants list
validating not in sync
etc.

That should work for eventually starting an unstartedNodes.
Do you agree?

Reply:
Yes, agree. I have test with `unstartedNodes` where I assign those at startup like this:
`SecurityGroup(self.getAllNodes() + self.unstartedNodes, self.eosioAccount, defaultNode=self.nodes[0])`
so that should work fine in my case